src/utils/validation.ts

```diff
@@ -1,8 +1,100 @@
-// TODO implement validation
+import { z } from 'zod';
+import { oisSchema, configSchema as airnodeConfigSchema } from '@api3/airnode-validator';
+import { configSchema as airkeeperConfigSchema } from '@api3/airkeeper/dist/src';
 import { OperationsRepository } from '../types';
 
-export type ValidationResult = [boolean, []];
+export const evmAddressSchema = z.string().regex(/^0x[a-fA-F0-9]{40}$/);
+export const evmBeaconIdSchema = z.string().regex(/^0x[a-fA-F0-9]{64}$/);
+export const evmTemplateIdSchema = z.string().regex(/^0x[a-fA-F0-9]{64}$/);
+export const evmEndpointIdSchema = z.string().regex(/^0x[a-fA-F0-9]{64}$/);
+export const evmXpubSchema = z.string().regex(/^xpub[a-zA-Z0-9]{107}$/);
```

Comment:
I'd actually say this is fine as is.

Reply:
I think it's appropriate for them to be in this file because these constants are specific to validation. I don't expect these to be used anywhere else. In fact they probably don't need to be exported either, but in the early phases of this dev having `export` on everything will reduce other development being blocked temporarily.